content/docs/deployment/csmoperator/modules/authorization v2.0 Tech Preview.md

```diff
@@ -0,0 +1,366 @@
+---
+title: Authorization v2.0 Tech Preview
+linkTitle: "Authorization v2.0 Tech Preview"
+description: >
+  Installing Authorization v2.0 Tech Preview via Dell CSM Operator
+---
+
+{{% pageinfo color="primary" %}}
+The CSM Authorization karavictl CLI is no longer actively maintained or supported. It will be deprecated in CSM 2.0.
+{{% /pageinfo %}}
+
+## Install CSM Authorization via Dell CSM Operator
+
+The CSM Authorization module for supported Dell CSI Drivers can be installed via the Dell CSM Operator.
+To deploy the Operator, follow the instructions available [here](../../#installation).
+
+### Prerequisite
+
+1. [Install Vault or configure an existing Vault](#vault-server-installation).
+
+2. Execute `kubectl create namespace authorization` to create the authorization namespace (if not already present). Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'authorization'. 
+
+3. Install cert-manager CRDs 
```

Comment:
For Operator, we fixed the issue for automating the cert-manager installation. Now it does not need to be done manually as a pre-req.

Reply:
Do the cert-manager manifests in the Operator include the cert-manager CRDs? I don't see that we have them.

Reply:
For now we leaving it here!